pkg/registry/apis/provisioning/jobs/sync/worker.go

```diff
@@ -437,12 +441,12 @@ func (r *syncJob) writeResourceFromFile(ctx context.Context, path string, ref st
 		Resource: parsed.GVR.Resource,
 		Group:    parsed.GVK.Group,
 	}
-
-	if r.resourcesLookup[id] {
-		result.Error = fmt.Errorf("duplicate resource name: %s", parsed.Obj.GetName())
+	existing, found := r.resourcesLookup[id]
+	if found {
+		result.Error = fmt.Errorf("duplicate resource name: %s, %s and %s", parsed.Obj.GetName(), path, existing)
```

Comment:
I changed from bool -> string so we could show the two files that target the same name